src/environments/assets.ts

```diff
@@ -1,4 +1,5 @@
 export const assets = [
+  'assets/release-notes-1.17.0-beta.0.json',
   'assets/release-notes-1.17.0-alpha.3.json',
   'assets/release-notes-1.17.0-alpha.2.json',
```

Comment:
Do we want to remove old pre-releases in favor of new ones? (I think this has not been discussed yet)

Reply:
I was thinking once the final release was out that might be a good time to do that?